_data/open.yml

```diff
@@ -8019,7 +8216,8 @@ concepts:
       en: multichoose
       property: fenced-stacked
       area: "combinatorics"
-      notation: "mrow ( mfrac $1 $2"
+      comments:
+       - "<math><mrow intent='multichoose($a1,$a2)'><mo mathvariant='bold'>(</mo><mfrac linethickness='0pt'><mi arg='a1'>n</mi><mi arg='a2'>n</mi></mfrac><mo mathvariant='bold'>)</mo></mrow></math>"
```

Comment:
you could replace the n with 𝐧 (U+1D427 (119847))?

Reply:
@physikerwelt  I  seemed to have two n there. I  made one an m but would you really use bold upright not just normal math italic?  but in general as I commented to Deyan I'd rather reviewing individual entries happens at w3c with more reasonable sized diffs as this PR was really asking about the change in format to add arg= attributes which is nearly completed now just 126 rows with `!!F` and no real mathml out of 1264 math elements. But diff for this PR is now nearly 5000 lines I want to merge as is,  to allow individual entries to be reviewed/edited at w3c (where more people have write access)